src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -884,7 +884,7 @@ protected function {$methodName}($lazyInitialization)
 
             if ($isProxyCandidate = $this->getProxyDumper()->isProxyCandidate($definition)) {
                 if (!$definition->isShared()) {
-                    $code .= sprintf('        %s = %1$s ?? ', $factory);
+                    $code .= sprintf('        %s ??= ', $factory);
```

Comment:
Can be replaced by, no?

```suggestion
                    $code .= '        %s ??= ';
```